playground/play001.ts

```
@@ -0,0 +1,46 @@
+        : Effect.succeed(a / b);
+    default:
+      throw new Error(`Unknown operation: ${op}`);
+  }
+};
+
+
+const addTwo = (a: number) => {
+  return (context: Context) => operationByContext(a + 2, context.factor, context.operation);
+};
+
+const provideDependencies = (f: Function, context: Context) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)